File: C/beginner/1070.c
// https://judge.beecrowd.com/pt/problems/view/1070
// Seis números ímpares

// Libraries
#include <stdio.h>

// Main Function
int main() {
    // Starting Code
    int usr_input;

    // Scanning user input
    scanf("%d", &usr_input);

    // Iterating
    for (int i = 0; i < 12; i++) {
        if ((usr_input + i) % 2 != 0) {
            printf("%d\n", usr_input + i);
        }
    }

    // End
    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra

File: Haskell/beginner/1070.hs
-- https://judge.beecrowd.com/pt/problems/view/1070
-- Seis Números Ímpares

-- Main body
main :: IO ()
main = do
    usr_value <- readLn :: IO Int
    let firstOdd = if odd usr_value then usr_value else usr_value + 1
    mapM_ print (take 6 [firstOdd, firstOdd + 2..])

-- By Fernando Serra
-- https://github.com/fernandosserra